packages/ag-charts-community/src/chart/keyboardUtil.ts

```diff
@@ -14,13 +14,19 @@ function computeCenter(bboxOrPath: Path | BBox | undefined) {
 
 type PickProperties = { bounds: Path | BBox | undefined; showFocusBox: boolean };
 
-export function drawPickedFocus(focusIndicator: FocusIndicator | undefined, pick: PickProperties) {
+function drawPickedFocus(focusIndicator: FocusIndicator | undefined, pick: PickProperties) {
     const { bounds, showFocusBox } = pick;
     if (showFocusBox) {
         focusIndicator?.updateBounds(bounds);
     }
 }
 
+export function getPickedFocusBBox({ bounds }: PickProperties): BBox {
+    if (bounds instanceof BBox) return bounds;
+    else if (bounds != null) return bounds.getBBox();
```

Comment:
nit: don't need the `else` since the previous block returns.

```suggestion
    if (bounds != null) return bounds.getBBox();
```